test_suites/global_bin/setup_fio.sh

```diff
@@ -6,20 +6,20 @@
 PATH=~/workspace/bin:/usr/sbin:/usr/local/bin:$PATH
 
 cd ~/workspace
```

Comment:
Please Sync your forked repo before creating 'pull request', so that we don't need to review the code from others. :)

Reply:
@yuhuijiang @yuxisun1217